src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Cognitive Services management functions for managing Microsoft Azure Cognitive Services accounts.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
@shahabhijeet , you mean both AssemblyVersion and AssemblyFileVersion need not to be changed or only AssemblyVersion need not to be changed? 
Note this PR introduced a breaking change (Location from property to method parameter) so we definitely need re-versioning.